ui/common/ContributionBar.tsx

```diff
@@ -129,7 +135,7 @@ export default function ContributionBar({
           const player = protagonists[idx]
           let { offsetAvatar, offsetBar } = protagonistState[idx]
           offsetBar = offsetBar ? `${offsetBar}px` : `-100%`
-          offsetAvatar = offsetAvatar ? `${offsetAvatar}px` : `-100%`
+          offsetAvatar = offsetAvatar ? `${offsetAvatar}px` : ``
```

Comment:
does this affect anything in the absence of the offestAvatar now being an empty string?

Reply:
I'll wait for this to be resolved b4 merging.